src/Option/README.md

```diff
@@ -0,0 +1,26 @@
+# ListBoxOption:
+
+Provides an ListBoxOption component that is passed as a child to the ListBox component.
+
+## API
+
+### `children` (node array; required)
+`children` specifies the array of elements that will be displayed within the ListBoxOption component.
+
+### `className` (string; optional)
+`className` specifies Bootstrap class names to apply to the ListBoxOption component. The default is an empty string.
+
+### `tag` (string; optional)
+`tag` is used to specify the element type of the rendered ListBoxOption component. The default is div. Example alternatives include li, Button, etc.
+
+### `onOptionSelect` (function, optional)
+`onOptionSelect` is a function that is called when an ListBoxOption component is selected, by mouse or by keyboard, to perform some additional action.
+
+### `other`
+`other` specifies any additional attributes on the ListBoxOption component, which can include aria-labelledby, id, key, etc.
```

Comment:
I think the reason I added this is because not all of our components have this behavior, and sometimes it can be very useful. The Button has it, and I only discovered it after looking through the source code. But I can remove it if it seems unnecessary.